gui/packages/ubuntupro/lib/pages/widgets/navigation_row.dart

```diff
@@ -32,10 +34,15 @@ class NavigationRow extends StatelessWidget {
           ),
         if (showNext) ...[
           const Spacer(),
-          ElevatedButton(
-            onPressed: onNext,
-            child: Text(nextText ?? lang.buttonNext),
-          ),
+          nextIsElevated
```

Comment:
I'd rather have this parameter name closer to the semantics, instead of reflecting an implementation detail. Something like: `nextIsSubmit` or `emphasizeNext` or swap its meaning and name it `equalEmphasis`, or come up with something better yet, can we?

Reply:
`nextIsAction`?